worker/containerdworker/networking.go

```diff
@@ -0,0 +1,89 @@
+package containerdworker
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/containerd/containerd"
+	"github.com/vishvananda/netlink"
+)
+
+//CreateBridgePair creats a veth to bridge the provided ethernet
+func CreateBridgePair(bridge string) (p *VethPair, err error) {
```

Comment:
Can we move this to a separate pkg, and support both workers?

Reply:
Got, from discussion of #62 , somehow, just focused on containerd. Will do it, so runc workers can use it.